docs/src/Module.doc.js

```diff
@@ -12,49 +12,201 @@ card(
+        description: 'Base Id used to give each item in the Module a unique Id',
       },
       {
         name: 'accessibilityExpandLabel',
+        href: 'expandable-default',
         type: 'string',
         required: true,
+        description:
+          'Label used to communicate to screen readers which module will be expanded when interacting with the title button. Should be something clear, like "Expand Security Policies Module"',
       },
       {
         name: 'accessibilityCollapseLabel',
+        href: 'expandable-default',
         type: 'string',
         required: true,
+        description:
+          'Label used to communicate to screen readers which module will be collapsed when interacting with the title button. Should be something clear, like "Collapse Security Policies Module"',
       },
       {
         name: 'items',
+        href: 'expandable-items',
         type:
           'Array<{| title: string, icon?: $Keys<typeof icons>, summary?: Array<string>, type?: "info" | "error", iconAccessibilityLabel?: string, children: ?React.Node |}>',
         required: true,
+        description:
+          'Array of modules to display in a stack - only one item can be expanded at a time',
       },
     ]}
   />
 );
 
 card(
   <Example
-    name="Example"
+    name="Non-Expandable"
+    description={`A Module is a container that can hold any content, and can optionally have a \`title\` that describes the content inside. The default, non-expandable Module is used to display information that should always be visible.`}
+    id="non-expandable-default"
+    defaultCode={`
+function ModuleExample() {
+  return (
+    <Flex direction="column" gap={2} maxWidth={800}>
+      <Module
+        id="ModuleExample - default - 1"
+        >
+        <Text size="md">This is example content.</Text>
+      </Module>
+      <Module
+        id="ModuleExample - default - 2"
+        title="Title"
+        >
+        <Text size="md">This is example content.</Text>
+      </Module>
+    </Flex>
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    name="Non-Expandable - Icon"
+    description={`
+    An Icon can be provided to be placed before the \`title\`.
+
+    It is recommended that icons be used sparingly to convey additional information, and instead should simply reenforce information in the title. If the icon does provide additional information about this module (locked, disabled, new, etc.), be sure to provide an \`iconAccessibilityLabel\`.
+    `}
+    id="non-expandable-icon"
+    defaultCode={`
+function ModuleExample() {
+  return (
+    <Box maxWidth={800} padding={2} column={12}>
+      <Module
+        id="ModuleExample - icon"
+        title="Title"
+        icon="lock"
+        iconAccessibilityLabel="Module Locked - check permission settings"
+        >
+        <Text size="md">This is example content.</Text>
+      </Module>
+    </Box>
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    name="Non-Expandable - Error"
```

Comment:
This example is a little odd to play around with…errors on an input should disappear when corrected, but in this example we continue to show the error module state even if the user enters something into the input. Perhaps input validation isn't a great example? Do we have real designs we can model this example on?

Also: should the error examples have a description to indicate when they should be used?

Reply:
Design guidelines still need to be written. There are no 'real examples' because no one has this component to use yet.

Reply:
I updated the examples to be fulling working, and we can add in more design guidance as we do the docs re-write (Ashley already has a v1 that will be iterated on)